### ISSUE TITLE
Update return types.

### ISSUE THREAD

**Issue by laurentmuller** [dustin10/VichUploaderBundle#1242]
### BC Break Report

|    Q        |   A
|------------ | ------
| BC Break    | No
| Version     | 1.18.0

#### Summary

I have a project that use this bundle with symfony version 5.4. When I run PHP unit tests. A warning is output:

`1x: Method "Symfony\Component\Form\DataTransformerInterface::reverseTransform()" might add "mixed" as a native return type declaration in the future. Do the same in implementation "Vich\UploaderBundle\Form\DataTransformer\FileTransformer" now to avoid errors or add an explicit @return annotation to suppress this message.`

To stop all this warnings, just modify code to add a return type. For example for FileTransformer warning:

Before:
```php
 /**
   * {@inheritdoc}
   */
  public function reverseTransform()
  {
      ...
  }
```
After:
```php
 /**
   * {@inheritdoc}
   */
  public function reverseTransform(): ?UploadedFile
  {
      ...
  }
```
Or
```php
 /**
   * {@inheritdoc}
   * 
   * @return UploadedFile|null
   */
  public function reverseTransform()
  {
      ...
  }
```

**Comment by garak** [comment]
Fixed in 35c80f2c5daf286abf1a04bc4f8e3eccabf7cd03

**Comment by laurentmuller** [comment]
Sorry, I didn't see this fix.
